firefox_puppeteer/api/appinfo.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class AppInfo(BaseLib):
+
+    @property
+    def browserTabsRemoteAutostart(self):
+        return self._get_property("browserTabsRemoteAutoStart")
+
+    def _get_property(self, prop_name):
+        with self.marionette.using_context('chrome'):
+            return self.marionette.execute_script("""
+            try {
+              return Services.appinfo[arguments[0]];
+            } catch (e) {
+              return null;
+            }""", script_args=[prop_name])
```

Comment:
Please indent the inner code by 2 blanks and put the last closing bracket of the JS code in its own line.